test/screenshotter/ss_data.yaml

```diff
@@ -309,7 +309,12 @@ TextStacked:
       \textsf{\textrm{\textbf{abc123}} \textbf{abc123} \textit{abc123}}\\
       \textit{abc123 \textbf{abc123} \textsf{abc123}}\\
   \end{matrix}
-TextWithMath: \text{for $a < b$ and $ c < d $}.
+TextWithMath: 
+  \begin{matrix}
+      \text{for $a < b$ and $ c < d $}. \\
+      \textsf{for $a < b$ and $ c < d $}. \\
+      \textsf{for $a < b \textbf{ and } c < d $}
```

Comment:
It would be good to have some tests for the behavior of old fonts.

Reply:
I'll update.